src/models/connectionProfile.ts

```diff
@@ -27,6 +27,7 @@ export class ConnectionProfile extends ConnectionCredentials implements IConnect
     public emptyPasswordInput: boolean;
     public azureAuthType: AzureAuthType;
     public azureAccountToken: string;
+    public expiresOn: number;
```

Comment:
shouldn't this also be `number | undefined` just like how it is in `ConnectionCredentials` ?